app/models/resource.py

```diff
@@ -5,11 +5,25 @@ class Resource(db.Model):
     __tablename__ = 'resources'
     id = db.Column(db.Integer, primary_key=True)
     name = db.Column(db.String(64))
-    address = db.relationship('Address', backref='resource', lazy='joined',
-                              uselist=False)
     description = db.Column(db.Text)
     website = db.Column(db.Text)
+    address_id = db.Column(db.Integer, db.ForeignKey('address.id'))
```

Comment:
`address_id = db.Column(db.Integer, db.ForeignKey('addresses.id'))`. The foreign key is a reference to a specific column in a table so it needs to be `addresses` not `address` because that's the actual table name.